Resources/public/js/collections.js

```diff
@@ -84,7 +85,11 @@
             }
 
             var html = this._getPrototypeHtml($prototype, this.internalCount++),
+            if (this.options.keepScripts) {
+                $row = $($.parseHTML(html, document, true));
```

Comment:
Could this be simplified to `$row = $($.parseHTML(html, document, this.options.keepScripts));` ?